.travis.yml

```diff
@@ -50,7 +50,7 @@ script:
   # We are using && below on purpose
   # ciPerformRelease must run only when the entire build has completed
   # This guarantees that no release steps are executed when the build or tests fail
-  - ./gradlew build idea -s && ./gradlew ciPerformRelease
+  - ./gradlew spotlessCheck && ./gradlew build idea -s && ./gradlew ciPerformRelease
```

Comment:
It could be probably implemented to fail fast in the build `./gradlew spotlessCheck build idea`, but that is simpler to achieve the same effect.

Reply:
I wanted to keep it clear that we fail fast on formatting only. We can probably later clean this up if we can enforce the correct dependency of the formatter and the build.